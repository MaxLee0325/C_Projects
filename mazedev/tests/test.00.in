Alpha
Charlie
Echo
Bravo
Delta
Foxtrot
END
Alpha Charlie
Charlie Delta
Alpha Bravo
Echo Charlie
Delta Bravo
Delta Foxtrot
Foxtrot Echo
END
Alpha
Charlie
Delta
Foxtrot
Echo
END
