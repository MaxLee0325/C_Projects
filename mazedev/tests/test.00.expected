Entering room Alpha.
Entering room Charlie.
Entering room Delta.
Entering room Foxtrot.
Entering room Echo.
Ended in room Echo.
